app/renderer/js/utils/domain-util.js

```diff
@@ -109,30 +106,9 @@ class DomainUtil {
 			return Promise.reject('This server has been added.');
 		}
 
-		domain = this.formatUrl(domain);
```

Comment:
We can't remove this line as then in line 110 `domain` used would be without formatting. So this line is required. (otherwise servers added without preceeding http/s would not be added).

Reply:
@abhigyank  i have added the line again.. Thanks